hnn_core/cell.py

```diff
@@ -293,18 +306,28 @@ def move_to_pos(self):
     # 2. a list needs to be created with a Dipole (Point Process) in each
     #    section at position 1
     # In Cell() and not Pyr() for future possibilities
-    def insert_dipole(self, yscale):
+    def insert_dipole(self, p_secs, dipole_sec):
```

Comment:
it's becoming clear that `p_secs` should be an attribute of `Cell` but I'm holding off on that in this PR.

Reply:
I don't see why `p_secs` needs to be an attribute of `Cell` since it isn't used anywhere outside `Cell.build()` and here. Since `Cell.insert_dipole()` is called after `build()`, why not just access the points via the NEURON Section objects?

Reply:
I'm imagining that `plot_cell_morphology` will move from `Network` to `Cell` and then `self.p_secs` will be used for the plot. I'm a little wary of relying on NEURON objects because of : 1) numerical issues we saw in the past 2) unpicklability. Since `p_secs` is picklable we can make it accessible to the user to inspect/modify etc.

Reply:
Fair enough!

Reply:
I agree that to enable `Cell.plot` we should keep HOC-objects out of ours for as long as possible.